arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+<map proto="1.4.2">
+<name>Bed Rush</name>
+<version>1.0.0</version>
+<phase>development</phase>
+<objective>Destroy the Bed to win the Round.  First Team to 5 Wins!</objective>
+<created>2024-07-01</created>
+<authors>
+    <author uuid="5e295775-a85d-467f-b041-bf817b28241c" contribution="Concept, Map"/> <!-- Lorzz -->
+    <author uuid="c25a6f7b-4c42-40da-8cd6-add53f0c84eb" contribution="XML, Map Aesthetics"/> <!-- arcadeboss -->
+</authors>
+<teams>
+    <team id="team-one" color="red" max="40">Red</team>
+    <team id="team-two" color="blue" max="40">Blue</team>
+</teams>
+<!-- observer spawn -->
+<spawns>
+    <default>
+        <regions>
+            <point yaw="90">20,34,0.5</point>
+        </regions>
+    </default>
+</spawns>
+<time result="score">15m</time>
+<world>
+    <timeset>18000</timeset>
+</world>
+<gamerules>
+    <doDaylightCycle>false</doDaylightCycle>
+</gamerules>
+<maxbuildheight>21</maxbuildheight>
```

Comment:
I feel like this might be too high, considering that you can build up to the enemy's spawn and jump there. I didn't see any regions that prevented camping there

Reply:
It could indeed be a problem, but this height is required to bridge from the spawn platform, which is how Lorzz has defined it.

We did consider and test out some spawn kill scenarios, but with the resistance entering players get figured it would balance out.

Would like to play test to see, but if this does become an issue, I'd prefer to go with an exclusion zone for invading team, but Lorzz didn't think it was needed.